lib/openshift/networkattachmentdefinition.rb

```diff
@@ -0,0 +1,9 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # represnets an Openshift Namespace
+  class Networkattachmentdefinition < ClusterResource
```

Comment:
Could you make Class name `NetworkAttachmentDefinition` and file name in snake case?

Reply:
For sure

Reply:
Done. thanks